taskiq/cli/worker/run.py

```diff
@@ -225,43 +261,42 @@ def run_worker(args: WorkerArgs) -> None:  # noqa: WPS213
     :param args: CLI arguments.
     """
     logging_queue = Queue(-1)  # type: ignore
-    listener = QueueListener(logging_queue, StreamHandler(sys.stdout))
-    basicConfig(
-        level=getLevelName(args.log_level),
-        format="[%(asctime)s][%(levelname)-7s][%(processName)s] %(message)s",
+    listener = QueueListener(logging_queue, logging.StreamHandler(sys.stdout))
+    logging.basicConfig(
+        level=logging.getLevelName(args.log_level),
+        format="[%(asctime)s][%(name)s][%(levelname)-7s][%(processName)s] %(message)s",
         handlers=[QueueHandler(logging_queue)],
     )
+    logging.getLogger("watchdog.observers.inotify_buffer").setLevel(level=logging.INFO)
     listener.start()
     logger.info("Starting %s worker processes.", args.workers)
 
-    global worker_processes  # noqa: WPS420
-
-    for process in range(args.workers):
-        work_proc = Process(
-            target=start_listen,
-            kwargs={"args": args},
-            name=f"worker-{process}",
-        )
-        work_proc.start()
-        logger.debug(
-            "Started process worker-%d with pid %s ",
-            process,
-            work_proc.pid,
-        )
-        worker_processes.append(work_proc)
+    action_queue: "Queue[Tuple[ProcessAction, Optional[int]]]" = Queue(-1)
```

Comment:
What about creating a class with event args? I think this will make the code easier to read.

For example :
```
class ReloadOneActionArgs:
    worker_num: int
```
or some common class for all events

Reply:
Maybe you're right. I'll refactor it a little more.

Reply:
Please check it. I created a separate class for managing processes and classes for all actions. Now it looks a bit better.